mlir/test/IR/locations.mlir

```diff
@@ -105,3 +105,10 @@ func.func @dialect_location() {
   test.attr_with_loc("dialectLoc" loc(#test.custom_location<"foo.mlir"*32>))
   return
 }
+
+// CHECK-LABEL: @location_attr
+// CHECK: test.op_with_loc_attr loc("cheetos":10:20)
+func.func @location_attr() {
+  test.op_with_loc_attr loc("cheetos":10:20)
```

Comment:
Can you add a test with the actual operation location specified and round-tripped? Reading this it is somehow ambiguous what is parsed/printed.

Reply:
Or add some discardable attributes afterward?

Reply:
Or multiple location in the op definition / format